micromanager_gui/main_window.py

```diff
@@ -14,14 +16,20 @@
 from qtpy.QtGui import QColor, QIcon
 from superqt.utils import create_worker, ensure_main_thread, signals_blocked
 
+try:
```

Comment:
not necessary.  you added to requirements (which I like)

Reply:
oh woops, I think I intended to make it optional. But if you're happy with requiring zarr that will make it simpler because I can just eliminate the `memmap` approach and only use zarr.